moderators.md

```diff
@@ -0,0 +1,143 @@
+
+## Enforcement
+
+[Moderators](governance.md#moderator) must understand [all R-multiverse policies](policies.md)
+and be prepared to enforce them.
+[Moderators](governance.md#moderator) must respond immediately and quickly to the following rare cases:
+
+1. If an R-multiverse package or [R-multiverse stakeholder](governance.md#r-multiverse-stakeholders) violates any applicable laws,
+contact the appropriate law enforcement agency.
+1. If an R-multiverse package on GitHub and violates any [GitHub policies](https://docs.github.com/en/site-policy),
+[report the violation to GitHub](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam).
+1. If an R-multiverse package on GitLab and violates any [GitLab policies](https://handbook.gitlab.com/handbook/legal/#gitlab-policies)
+such as [Acceptable Use](https://handbook.gitlab.com/handbook/legal/acceptable-use-policy/),
+[report the violation to GitLab](https://docs.gitlab.com/ee/user/report_abuse.html).
+1. In case of an extreme policy violation, or repeated violations of the [code of conduct](conduct.md),
+ block the user from [R-multiverse GitHub repositories](https://github.com/r-multiverse)
+ by adding the GitHub username to the [list of blocked users](https://github.com/organizations/r-multiverse/settings/blocked_users).
+1. If an R-multiverse package has a security vulnerability such as malware,
+report all relevant versions of the package to the [R Advisory Database](https://github.com/rconsortium/r-advisory-database).
+
+If a package violates any R-multiverse [policy](policies.md)
+(for example, if the [license](https://en.wikipedia.org/wiki/Free_and_open-source_software) is invalid
+^[[This list](https://github.com/r-multiverse/community/blob/main/nonstandard_licenses.json) automatically track packages
+without valid [FOSS](https://en.wikipedia.org/wiki/Free_and_open-source_software) licenses.
+[Moderators](governance.md#moderator) should block each package in this list
+until the package authors add a valid [FOSS](https://en.wikipedia.org/wiki/Free_and_open-source_software) license
+and resolve all other [policy](policies.md) violations.]),
+a [moderator](governance.md#moderator) must block it from R-multiverse:
+
+1. Locate the [package listing](https://github.com/r-multiverse/contributions/tree/main/packages)
+in the [contributions repository](https://github.com/r-multiverse/contributions).
+1. Replace the URL or custom JSON in the listing with free-form text (not a URL or parseable JSON) explaining why the package was blocked.
+^[If the listing cannot be parsed as a URL or JSON, then a bot will automatically remove the package from R-multiverse infrastructure.]
+1. Contact the package authors (confidentially if needed) to inform them of these actions.
+
+# Contact
+
+If you would like to be a [moderator](governance.md#moderator),
+please reach out to one of the [current Administrators](team.md#administrators).
```

Comment:
should we have a group email address? which wouldn't prevent people from reaching out first to the one administrator they know best to ask questions.

I would prefer we do not merge this PR before we have a volunteering *form* (whose questions we'd make available outside of the form for those preparing answers in advance) as a free-text volunteering email is stressful and rather unfair as some people are better at this without this meaning they'd be better/worse moderators.

Reply:
> should we have a group email address? 

I'm not sure. I actually like not having one because it nudges more of the communication out in the open.

> I would prefer we do not merge this PR before we have a volunteering form

Happy to write one. How are these forms typically deployed?

Reply:
At rOpenSci we use Airtable, maybe (probably) the free plan could be enough for this. https://airtable.com/pricing

Alternatively there's Google Forms but not everyone likes Google.

These are just the ones I know.

Reply:
Google Forms looks more convenient and easy to use. Do you have a preference?

Reply:
Added 88c53cd2feec51b2968e866bd2589c962ea8d82a to propose questions for the web form. I can create the form itself when we agree to merge this PR.

Reply:
> We could include a table of metadata on moderators there: hours volunteered and areas of expertise.

Maybe we can avoid recording hours in a separate place? And track expertise and affiliations at https://r-multiverse.org/team.html?